client/js/gdpr/chart.js

```diff
@@ -21,8 +21,11 @@ export class GdprChart extends HTMLElement {
 
     this.hoverState = {seriesIndex: null, x: null, y: null};
 
+    this.mainContainer = document.createElement('div');
+    this.mainContainer.id = 'main-container';
+
     this.scrollContainer = document.createElement('div');
-    this.scrollContainer.classList.add('chart-scroll-container');
+    this.scrollContainer.id = 'scroll-container';
```

Comment:
is there a reason why changing the class `chart-scroll-container` into the id `scroll-container` is a better design choice?

Reply:
Not really; I accidentally ended up implementing the same functionality twice b/c I was switching branches so much. Both styles are functionally equivalent, but I figured I might as well use an ID since this element is unique within the shadow root.